packages/quantic/force-app/main/default/lwc/quanticFacet/quanticFacet.js

```diff
@@ -96,19 +129,64 @@ export default class QuanticFacet extends LightningElement {
     return this.values.length !== 0;
   }
 
+  get isAnyChecked() {
+    return this.state.hasActiveValues;
+  }
+
+  get hasSearchResults() {
+    return this.facet.state.facetSearch.values.length !== 0;
+  }
+
+  get facetSearchResults() {
+    const results = this.facet.state.facetSearch.values;
+    return results.map((result) => ({
+      value: result.rawValue,
+      state: 'idle',
```

Comment:
what is that state idle?

Reply:
The `facetSearch.search()` only returns facet values that have not already been selected so I assumed it was safe to set the states of all facetSearch results as `idle` rather than `selected`

Reply:
Do the results have a `.state` ? if yes I would use that. Otherwise you can leave it, it's just puzzling if you see the code and you don't know that information, you wonder why 'idle'

Reply:
Yes sorry it's a bit confusing. Without search, the values displayed in the facets have type `FacetValue`, which has a `.state`, but the values returned from `facetSearch.values` have type `SpecificFacetSearchResult`, which don't have a `.state`. 
Here I convert `SpecificFacetSearchResult` to `FacetValue`, so that selecting a value from facetSearch still works as intended (if left as `SpecificFacetSearchResult` it would sometimes do nothing on selection). That is why I set `state: 'idle'` there.
Hope that makes things a little clearer 😄

Reply:
ah yes okay!